internal/workspace_client_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestUcAccWorkspaceClient_CurrentWorkspaceId(t *testing.T) {
+func TestUcAccWorkspaceClient_CurrentWorkspaceId_NoTranspile(t *testing.T) {
```

Comment:
Curious, what's the reason for this? From the comment:
````
// Tests with NoTranspile suffix are too expensive to automatically
// translate, we just ignore them.
````
I guessing that making it transpile friendly would be more effort?

Reply:
Yeah, basically that.